script/release

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+
+rm -rf pkg/*
+bundle exec rake build
+gem push pkg/*
```

Comment:
I think you'll need to put a `$@` on the last line so we can pass in the `--key` param without us having to store it in the repository here: 

```bash
gem push pkg/* $@
```

and then in the jenkins script:
```bash
# ...
script/release --key foobar
```

I changed the default key to an intentionally wrong value to hopefully prevent anyone from accidentally pushing gems to rubygems.org, so we need to specify the key name. I"ll IM you that info